universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -284,10 +298,12 @@ private DomContent formQuestionTypeSelect(QuestionType selectedType) {
    * question definition's repeater id.
    */
   private SelectWithLabel repeaterOptionFromQuestionDefinition(
-      QuestionDefinition questionDefinition) {
+      QuestionDefinition questionDefinition, Optional<String> maybeRepeaterName) {
```

Comment:
It seems a little funny to me that this method ultimately still needs the questionDefinition _and_ the maybeRepeaterName. Makes me think we should should compute the name here instead of piping it through.

Do we do this because getting the name requires access to the service?

Reply:
Computing the name requires access to the service, which is why it is computed way high up, and this optional string is passed through.

Wec could do the `String displayString = maybeRepeaterName.orElse(NO_REPEATER_DISPLAY_STRING)` earlier.

Reply:
Cool cool. The suggested change makes sense to me, but don't feel strongly.